src/Reflectivity-Tests/VariableBreakpointMockClass.class.st

```diff
@@ -36,8 +36,7 @@ VariableBreakpointMockClass >> methodWithTempsAndArg: arg [
 ]
 
 { #category : #accessing }
-VariableBreakpointMockClass >> v1 [
-	^ v1
+VariableBreakpointMockClass >> v1 [ ^v1
```

Comment:
Is this formatting normal?

Reply:
Some test is rewriting the method (without formatting?). So I committed the result because I was tired of unselecting it at each commit. Cf #13351